pkg/provider/provider.go

```diff
@@ -191,11 +191,11 @@ func buildTestEnvironment() { //nolint:funlen
 	}
 	env.OpenshiftVersion = data.OpenshiftVersion
 	env.K8sVersion = data.K8sVersion
-	helmchartreleases := data.HelmChartReleases
-	for _, raw := range helmchartreleases {
-		for _, helm := range raw {
-			if !isSkipHelmChart(helm.Name, data.TestData.SkipHelmChartList) {
-				env.HelmChartReleases = append(env.HelmChartReleases, helm)
+	helmChartReleases := data.HelmChartReleases
```

Comment:
I don't think you need this extra helper variable as it won't be used anywhere else. You can just do:
```
for _, nsHelmChartReleases := range data.HelmChartReleases {
    ...
```